valhalla/thor/triplegbuilder.h

```diff
@@ -24,16 +24,6 @@ namespace thor {
  */
 class TripLegBuilder {
 public:
-  /**
```

Comment:
i have no idea why this object was being instantiated. there was actually no state at all in any of the class so i changed it to be all static so that, from the outside it looks like what it is, a collection of static functions.